certs/openssl.server.conf

```diff
@@ -1,4 +1,4 @@
 extendedKeyUsage = serverAuth
-subjectAltName = DNS:*.marinade.finance,DNS:localhost,IP:0.0.0.0,IP:127.0.0.1
+subjectAltName = DNS:*.rpcpool.wg,DNS:localhost,IP:0.0.0.0,IP:127.0.0.1
```

Comment:
we dont need a wildcard here? cant we move these scripts into the ansible isntead.